aiogram/bot/bot.py

```diff
@@ -176,6 +176,24 @@ async def get_me(self) -> types.User:
         result = await self.request(api.Methods.GET_ME, payload)
         return types.User(**result)
 
+    async def log_out(self) -> types.User:
```

Comment:
return annotation should be false

Reply:
fixed 
https://github.com/aiogram/aiogram/pull/454/commits/ee523afb8c2c7ba18987d8f6982584d548e167e8